js/interface.js

```diff
@@ -2522,6 +2522,15 @@ var DynamicLists = (function() {
 
       return Promise.all([basePromise, loopPromise, detailPromise, filterLoopPromise, otherLoopPromise, cssPromise, jsPromise]);
     },
+    toggleTabVisibility: function(id, isInvisible, fromReset) {
+      var $el = document.getElementById(id);
```

Comment:
Not a jQuery object. Doesn't need `$`.

```suggestion
      var el = document.getElementById(id);
```